app/api/deps.py

```
@@ -0,0 +1,65 @@
+from app.infrastructure.clients import KutisClient, LmsClient
+from app.infrastructure.scrapers import KutisScraper, LmsScraper
+from app.infrastructure.exceptions import ClientException
+from app.schemas.students.request import StudentTimetableRequest
+from app.usecases.students import (
+    StudentAuthenticationUseCase,
+    StudentInformationUseCase,
+    StudentTimetableUseCase,
+)
+
+
+async def get_client(
```

Ruff (0.11.9)

Missing return type annotation for public function `get_client`

(ANN201)